src/filesystem/implementation.js

```diff
@@ -441,6 +441,21 @@ function make_directory(context, path, callback) {
   find_node(context, path, check_if_directory_exists);
 }
 
+/**
+ * access_file
+ */
+function access_file(context, path, mode, callback) {
+  path = normalize(path);
+  find_node(context, path, function (err) {
+    if (err) {
+      return callback(err);
+    }
+    // TO DO: we currently only support Constants.fsConstants.F_OK
```

Comment:
Update this comment to also include a reference to the Issue I filed https://github.com/filerjs/filer/issues/561.

nit: change `TO DO` to `TODO` (no space)